### PR TITLE
Realtime cookbook

### PR DESCRIPTION
**Pull request by jonny22094**

### CODE REVIEW

File: docs/cookbook/web/solutions/real-time-chat.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+# Real-time chat
+
+**To get started you need:**
```
Comment (mkucharz) [inline docs/cookbook/web/solutions/real-time-chat.md]: Please make it in the same format as other cookbooks:
- Preparation time etc.
- Problem to solve
- Solution

File: docs/cookbook/web/solutions/real-time-chat.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+## Create instance and socket
+
+  You must add to your project syncano cli and create instance.
+  
+  ```bash
+    npm install @syncano/cli
+    npx s init
+  ```
+
+  Now you are ready to create new socket.
```
Comment (mkucharz) [inline docs/cookbook/web/solutions/real-time-chat.md]: `Socket` with capital letter

File: docs/cookbook/web/solutions/real-time-chat.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+  ```bash
+    npx s create chat
+  ```
+
+## Create real-time channel
+
+We must declare 2 sockets.
```
Comment (mkucharz) [inline docs/cookbook/web/solutions/real-time-chat.md]: Sockets

File: docs/cookbook/web/solutions/real-time-chat.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+          message:
+            type: string
+            description: Send message
+            examples:
+              - Hello!!
+      globalChannel:
+        channel: global-message
+  ```
+
+  In your socket to send message you must declare:
```
Comment (mkucharz) [inline docs/cookbook/web/solutions/real-time-chat.md]: Socket

File: docs/cookbook/web/solutions/real-time-chat.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+
+      channel.publish("global-message", args)
+    }
+  ```
+
+## Create client
+
+Ok, now you're ready to listen on created channel in your project.
+
+You must import syncano client to your project
```
Comment (mkucharz) [inline docs/cookbook/web/solutions/real-time-chat.md]: Syncano

File: docs/cookbook/web/solutions/real-time-chat.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+      // Create new li with recived message in html ul
+      mesagesContainer.appendChild(mess);
+      // Scroll your messages container to bottom 
+      mesagesContainer.scrollTop = mesagesContainer.scrollHeight;
+    })
+```
+
+
+
+
```
Comment (mkucharz) [inline docs/cookbook/web/solutions/real-time-chat.md]: Too much empty lines.

File: docs/cookbook/web/solutions/real-time-chat.md [hunk 7]
```diff
@@ -1,28 +1,35 @@
...
+- Preparation: **10 minutes**
+- Requirements:
+  - Initiated Syncano project
+  - Basic understanding of JS, HTML & CSS
+  - How Socket works in Syncano
 
+### Problem to solve
 
-## Create instance and socket
+You want to build simple real-time chat
```
Comment (mkucharz) [inline docs/cookbook/web/solutions/real-time-chat.md]: Missing dot.

File: docs/cookbook/web/solutions/real-time-chat.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+    .addEventListener('message', message => {
+      // In payload we hold received message
+      const {payload} = JSON.parse(message.data);
+
+      // Create new html tag and inject message content to it
+      const mess = document.createElement('li');
+            mess.innerHTML = payload.message;
+
+      // Create new li with recived message in html ul
+      mesagesContainer.appendChild(mess);
```
Comment (PetKatt) [inline docs/cookbook/web/solutions/real-time-chat.md]: podwójne "ss", messages?

**Pull request merged**